src/bp/ui-studio/src/web/views/Content/index.tsx

```diff
@@ -82,25 +82,37 @@ class ContentView extends Component<Props, State> {
     this.props.contentItems.forEach((element: ContentElementUsage) => {
       element.usage = []
       Object.values(this.props.flows.flowsByName).forEach((flow: FlowView) => {
-        flow.nodes.forEach((node: NodeView) => {
-          const usage: ContentUsage = {
-            type: 'Flow',
-            name: flow.name,
-            node: node.name,
-            count: 0
-          }
+        // Skip skill flows
+        if (!flow.skillData) {
+          flow.nodes.forEach((node: NodeView) => {
+            const usage: ContentUsage = {
+              type: 'Flow',
+              name: flow.name,
+              node: node.name,
+              count: 0
+            }
 
-          const addUsage = (v: string | ActionBuilderProps) => {
-            if (typeof v === 'string' && v.startsWith(`say #!${element.id}`)) {
-              if (!usage.count) {
-                element.usage.push(usage)
+            const addUsage = (v: string | ActionBuilderProps) => {
+              if (typeof v === 'string' && v.startsWith(`say #!${element.id}`)) {
+                if (!usage.count) {
+                  element.usage.push(usage)
+                }
+                usage.count++
               }
-              usage.count++
             }
-          }
-          node.onEnter?.forEach(addUsage)
-          node.onReceive?.forEach(addUsage)
-        })
+
+            if (node.flow && node.type && node.type === 'skill-call' ) {
```

Comment:
you can use ts optional chaining here instead. 

`node.flow && node.?type === 'skill-call'`

just a bit less verbose

Reply:
You mean `node.flow && node.type? === 'skill-call'`? Anyways, the `node.type` validation is unnecessary, so I'll simply remove it!